calibrators/docs/tutorials/marker_radar_lidar_calibrator.md

```diff
@@ -0,0 +1,237 @@
+
+## Launching the tool
+
+In this tutorial, we take the X2 project as an example.
+First, run the sensor calibration manager:
+
+```text
+ros2 run sensor_calibration_manager sensor_calibration_manager
+```
+
+In `project`, select `x2`, and in `calibrator`, select `marker_radar_lidar_calibrator`. Then, press `Continue`.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/menu1.jpg" alt="menu1">
+</p>
+
+A menu titled `Launcher configuration` should appear in the UI, and the user may change any parameter he deems convenient.
+For this tutorial, we modify the default value `radar_name` from `front_left` to `front_center`. After configuring the parameters, click `Launch`.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/menu2.jpg" alt="menu2">
+</p>
+
+The following UI should be displayed. When the `Calibrate` button becomes available, click it.
+If it does not become available, it means that either the required `tf` or services are not available. In this case, since the `tf` are published by the provided rosbag, run the bag with the command `ros2 bag play radar_lidar.db3 --clock -r 0.5` and click the `Calibrate` button.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/menu3.jpg" alt="menu3">
+</p>
+
+### Extract background model
+
+Once the user starts running the tutorial rosbag, the pointcloud should appear in `rviz` as shown in the example below. Press the `Extract Background Model button` in the UI to start extracting the background.
+
+Note that the user should remove the radar reflector from the calibration area before clicking the button. Ideally, there should not be any moving objects, such as humans or vehicles, in the calibration area.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/rviz1.jpg" alt="rviz1" width="500">
+</p>
+
+Once the user clicks the button, the result will appear as shown in the image below.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/rviz2.jpg" alt="rviz2" width="500">
+</p>
+
+Once the background is extracted, the result will appear as shown in the image below. The user can see that the `Add lidar-radar pair` button is enabled.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/rviz3.jpg" alt="rviz3" width="500">
+</p>
+
+Also, the following text should be shown in the console.
+
+```text
+[marker_radar_lidar_calibrator]: Finished background model initialization
+```
+
+Additionally, the user can click the topics: `lidar_background_pointcloud`, `lidar_foreground_pointcloud`, `radar_foreground_pointcloud` and `radar_background_pointcloud` on the left bar of the `rviz` to visualize the output of background and foreground pointcloud. In the [documentation](../../marker_radar_lidar_calibrator/README.md), we referred to `radar foreground objects`; here, we call it radar_foreground_pointcloud for visualization purposes.
```

Comment:
One instance of radar_foreground_pointcloud lacks `radar_foreground_pointcloud`

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/52ac038f543fbcfddbdde06cad153ab4ffecf788